app/lib/forem_stats_driver.rb

```diff
@@ -0,0 +1,28 @@
+class ForemStatsDriver
+  def self.select_driver
```

Comment:
That's fine. It exists so that when there is more than just one driver implementation (which will be in the next PR), the code can just figure out from the configuration where to send stats. i.e. is there a New Relic API Key? If so, then use the New Relic driver, with the Datadog driver always being the default.